res/i18n/fr/labels.json

```diff
@@ -339,7 +339,7 @@
 	},
 	
 	"mute": "Muet",
-	"muting": "Silencier",
+	"muting": "Rendre muet",
```

Comment:
I think that we need to get a (direct and visual) relation between the translation of "mute" and "muted", for get an easyer and faster usage of the GUI, so i think that "Rendre muet" is good.
But i'm OK, "Silencier" is a good academic word.

Reply:
So what do we do ? Academic way or simpler and understandable way ?
Maybe one word @sadko4u ?

Reply:
I don't understand the french at all. All what i can do - just refer to the documentation which describes the behaviour of the corresponding control:

```
Muting - when enabled, turns off sample playback for selected instrument when the Channel Control MIDI message is received.
```
https://lsp-plug.in/?page=manuals&section=multisampler_x12

So please refer to the control in term `what it does` instead of `what it is` when translating.
All misunderstanding can be simply solved by just reading the corresponding manual page for the corresponding plugin.